m2cgen/interpreters/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .java.interpreter import JavaInterpreter
+from .python.interpreter import PythonInterpreter
+
+__all__ = [
+    JavaInterpreter,
+    PythonInterpreter
+]
```

Comment:
Looks like the only place where we could use this code is in exporters.py - and we don't. So I don't think we need this code.

Reply:
Don't we use it in "exporters.py"?

Reply:
My bad!